outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "noop" {

```

Comment:
Does this explicitly say there are no outputs from this module?

Reply:
If a capability has no outputs, merging capability outputs in the app module will cause TF to error.
This adds an output that is never used to prevent TF from erroring.